src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -17,6 +19,8 @@ export class NavbarComponent implements OnInit {
   isNavbarCollapsed = true;
   swaggerEnabled?: boolean;
   version: string;
+  account: Account | null = null;
```

Comment:
null hogy lehet???? az nem normalis

Reply:
A Jhipster szinte mindenhol igy használja az Accountot ha megprobálom átírni másra akkor ezt az errort adja: "TS2322: Type 'Account | null' is not assignable to type 'string | Account'. Type 'null' is not assignable to type 'string | Account'." Igy ezért használtam igy is és jól müködik.